releng_sop/koji_create_package_in_release.py

```diff
@@ -102,6 +102,12 @@ def get_parser():
         help="PDC release ID, for example 'fedora-24', 'fedora-24-updates'.",
     )
     parser.add_argument(
+        "owner",
+        metavar="OWNER",
+        default=argparse.SUPPRESS,
+        help="Package owner",
```

Comment:
A period might be good at the end of this, to stay consistent with the other help texts.